### PR TITLE
zigzag traversal in a binary tree

### PR DESCRIPTION
**Pull request by SakshiiAgrawal**
The idea is to use two stacks. We can use one stack for printing from left to right and other stack for printing from right to left. In every iteration, we have nodes of one level in one of the stacks. We print the nodes, and push nodes of next level in other stack.

I am creating a pull request for...

- [yes ] New algorithm
- [ ] Update to an algorithm
- [x] Fix an error
- [ ] Other - *Describe below*

### CODE REVIEW

File: data-structures/zigzagtraversal_iterative.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            if temp.right:  
+                s2.append(temp.right)
+  
+        # Print nodes of current level from s2 and append nodes of next level to s1  
+        while not len(s2) == 0: 
+            temp = s2[-1]  
+            s2.pop()  
+            print(temp.data, end = " ")  
+  
+            # Note that is rightt is appended before left
```
Comment (hemanth-kotagiri) [inline data-structures/zigzagtraversal_iterative.py:54]: Just one small change. ``rightt`` to ``right``. Writing well-documented code is extremely important. Try fixing any other issues of the same kind. Code LGTM.

**Review by hemanth-kotagiri — CHANGES_REQUESTED** [review]
LGTM.

File: data-structures/zigzagtraversal_iterative.py (hunk 1 above)
Comment (SakshiiAgrawal) [reply data-structures/zigzagtraversal_iterative.py:54]: sure, will do it

**Review by abranhe — APPROVED** [review]
Thanks so much for your contribution to [The All ▲lgorithms Project](https://github.com/allalgorithms). Without people like you submitting Pull Requests we couldn't run this project. You rock!

**Pull request merged**